specifications/api/swagger.yaml

```diff
@@ -4748,6 +4748,17 @@ x-errBalanceTxEraNotSupported: &errBalanceTxEraNotSupported
       type: string
       enum: ["balance_tx_era_not_supported"]
 
+x-errTxNotInNodeEra: &errTxNotInNodeEra
+  <<: *responsesErr
+  title: tx_not_in_node_era
+  properties:
+    message:
+      type: string
+      description: The transaction could be deserialised, just not in the era of the local node.
```

Comment:
Further to previous comments, I wonder if it's possible to make the direction of time inequality explicit here. ^^^

(Both in the name of the error, and the description.)

For example, I'm guessing that this error cannot be produced in a situation where the node era is later than the transaction era?